src/test/java/tests/workers/imports/CucumberHandlerTest.java

```diff
@@ -0,0 +1,61 @@
+package tests.workers.imports;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import main.model.db.imports.ImportHandlers.Cucumber;
+import main.model.db.imports.ImportHandlers.NUnitV3;
+import main.model.db.imports.TestNameNodeType;
+import main.model.dto.*;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import utils.FileUtils;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.TimeZone;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+
+public class CucumberHandlerTest {
+    private Cucumber cucumber;
+    private DtoMapperGeneral mapper = new DtoMapperGeneral();
+
+    @BeforeMethod
+    public void tryParse(){
+        try {
+            String string = "January 2, 2010";
+            DateFormat format = new SimpleDateFormat("MMMM d, yyyy", Locale.ENGLISH);
+            format.setTimeZone(TimeZone.getTimeZone("UTC"));
+            cucumber = new Cucumber(FileUtils.getResourceFile("reports/cucumber/cucumber.json"), format.parse(string));
+        } catch (Exception e){
+            assertNull(e, String.format("Failed on Handler Creating: %s", e.getMessage()));
```

Comment:
You can use Assert.fail(String.format("Failed on Handler Creating: %s", e.getMessage()), e);